erp/provider/geocoder.py

```diff
@@ -32,6 +32,7 @@ class Provider(StrEnum):
 ]
 
 
+# TODO check if this can be deleted
```

Comment:
@mlvernay I am not sure how we could end up in this code / if that was even possible. We can discuss it next week. I removed the final usage of this function, only a few tests are left. Let me know if:
- You are ok with me removing this (I can show you my doubts about the usage)
- You would prefer this to be removed here or in a separate PR